rustls/src/webpki/client_verifier_builder.rs

```diff
@@ -43,6 +50,20 @@ impl ClientCertVerifierBuilder {
         self
     }
 
+    /// When CRLs are provided with [`with_crls`][Self::with_crls] only check the end entity
```

Comment:
Maybe tighten up the docstring headline a bit? (I'd start by removing ", not the entire chain", which feels like something that doesn't need to be in the headline.)

Reply:
Done. Hopefully clearer now :crossed_fingers: